dev/tests/js/jasmine/tests/app/code/Magento/Customer/frontend/js/customer-data.test.js

```diff
@@ -3,30 +3,34 @@
  * See COPYING.txt for license details.
  */
 
-/*eslint-disable max-nested-callbacks*/
-/*jscs:disable jsDoc*/
+/* global _ */
+/* eslint max-nested-callbacks: 0 */
+/* jscs:disable jsDoc*/
 
 define([
+    'squire',
     'jquery',
-    'underscore',
```

Comment:
Seems like we should add it back as it's used in the code below

Reply:
Breaks following tests:

`"init" method` -> `Calls "reload" method when expired sections do not exist`
```
Expected spy reload to have been called. (1)
```

`"set" method` -> `Does not throw before component is initialized`
```
Expected function not to throw, but it threw TypeError: undefined is not an object (evaluating 'sectionData['data_id']'). (1)
```

Reply:
Was able to figure out how to load underscore via Squire and that resolves the issues.